generator/dart/generated/google_cloud_protobuf/lib/protobuf.dart

```diff
@@ -125,6 +125,31 @@ class Duration extends Message {
   }
 }
 
+/// A generic empty message that you can re-use to avoid defining duplicated
+/// empty messages in your APIs. A typical example is to use it as the request
+/// or the response type of an API method. For instance:
+///
+///     service Foo {
+///       rpc Bar(google.protobuf.Empty) returns (google.protobuf.Empty);
```

Comment:
For a future PR (maybe). In many languages the return type for such RPCs becomes some kind of "unit type", such as `void` or `()` (in Rust) .

Reply:
Gotcha! We do that currently for RPC responses (see https://github.com/googleapis/google-cloud-rust/blob/main/generator/internal/dart/annotate.go#L667 and https://github.com/googleapis/google-cloud-rust/blob/main/generator/testdata/dart/protobuf/golden/secretmanager/lib/secretmanager.dart#L88). While working on LRO support however the generic type params got pretty complicated, and being able to use a generated `Empty` class may simplify things (even if not being 100% idoimatic).